src/styles/calendar.scss

```diff
@@ -0,0 +1,59 @@
+.calendar {
+  padding: $block-padding;
+
+  &__days {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: $cell-size * 7 + $cell-gap * 6 + $block-padding * 2;
+    margin: auto;
+    gap: $cell-gap;
+  }
+
+  &__day {
+    position: relative;
+    list-style: none;
+    height: $cell-size;
+    width: $cell-size;
+    border: 1px solid black;
+    background-color: #eee;
+    transition-duration: 0.5s;
+
+    &:hover {
+      background-color: #ffc0cb;
+      transform: translate(0, -20px);
+    }
+
+    &::before {
+      position: absolute;
+      font-size: 30px;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+    }
+  }
+
+  //number of day
+
+  @for $nth-day from 1 to 32 {
+    &__day:nth-child(#{$nth-day})::before {
+      content: "#{$nth-day}";
+    }
+  }
+
+  //setting witch day gonna be first in the month
+  
+  $day-week:
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/120621793-e30e3700-c466-11eb-9810-28c915ad8c6a.png)
I can't see this modifier